parachain/pallets/verifier-lightclient/src/mock.rs

```diff
@@ -67,13 +72,49 @@ impl system::Trait for MockRuntime {
 	type SystemWeightInfo = ();
 }
 
+impl system::Trait for MockRuntimeWithPoW {
+	type BaseCallFilter = ();
+	type Origin = Origin;
+	type Call = ();
+	type Index = u64;
+	type BlockNumber = u64;
+	type Hash = H256;
+	type Hashing = BlakeTwo256;
+	type AccountId = AccountId;
+	type Lookup = IdentityLookup<Self::AccountId>;
+	type Header = Header;
+	type Event = MockEvent;
+	type BlockHashCount = BlockHashCount;
+	type MaximumBlockWeight = MaximumBlockWeight;
+	type DbWeight = ();
+	type BlockExecutionWeight = ();
+	type ExtrinsicBaseWeight = ();
+	type MaximumExtrinsicWeight = MaximumBlockWeight;
+	type MaximumBlockLength = MaximumBlockLength;
+	type AvailableBlockRatio = AvailableBlockRatio;
+	type Version = ();
+	type ModuleToIndex = ();
+	type AccountData = ();
+	type OnNewAccount = ();
+	type OnKilledAccount = ();
+	type SystemWeightInfo = ();
+}
+
+
```

Comment:
Can we move the parameter types down here from above:

```suggestion
parameter_types! {
	pub EthashDisabledConfiguration: EthashConfiguration = EthashConfiguration { verify_pow: false };
	pub EthashEnabledConfiguration: EthashConfiguration = EthashConfiguration::default();
}
```